service/src/main/java/bio/terra/profile/service/profile/flight/create/CreateProfileFlight.java

```diff
@@ -56,6 +58,7 @@ public CreateProfileFlight(FlightMap inputParameters, Object applicationContext)
       // we can link the profile to the MRG only after the Sam resource has been created
       addStep(new LinkBillingProfileIdToMrgStep(samService, billingProfile, user));
     }
-    addStep(new CreateProfileFinishStep());
+    addStep(new RecordProfileCreationStep(changeLogDao));
+    addStep(new CreateProfileFinishStep(changeLogDao));
```

Comment:
Both of the above steps look like they're recording a profile creation event, is that intentional?

Reply:
Nope - I just forgot to remove the dao from the `CreateProfileFinishStep` constructor, when I moved the rest of the logic to `RecordProfileCreationStep`.